example/lib/main.dart

```diff
@@ -246,6 +253,21 @@ class _MyAppState extends State<MyApp> {
               mainAxisAlignment: MainAxisAlignment.center,
               crossAxisAlignment: CrossAxisAlignment.center,
             ),
+            Container(
+              width: 56.0,
```

Comment:
Probably delete this so it can fill the dimensions of your device. Add margins to make it look nice.